pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -126,9 +126,9 @@ fi
 
 if [[ "$isCpp" = 1 ]]; then
     if [[ "$cppInclude" = 1 ]]; then
-        NIX_@infixSalt@_CFLAGS_COMPILE+=" ${NIX_@infixSalt@_CXXSTDLIB_COMPILE-@default_cxx_stdlib_compile@}"
+        NIX_@infixSalt@_CFLAGS_COMPILE+=" ${NIX_@infixSalt@_CXXSTDLIB_COMPILE:-@default_cxx_stdlib_compile@}"
```

Comment:
The missing `:` might have been a bug: since the variable is declared in `add-flags.sh` the default substitution will not come into play with `-`, only with `:-`.